ion/writer.go

```diff
@@ -166,3 +174,24 @@ func (w *writer) clear() {
 	w.fieldName = ""
 	w.annotations = nil
 }
+
+// Resolve resolves a symbol to its ID.
+func (w *writer) resolve(api, sym string) (uint64, error) {
```

Comment:
In the context of symbols, we usually use the verb "resolve" to mean the inverse: given some symbol ID, look up the corresponding text. Could you rename this? (`findOrCreateSid`, maybe?)